.github/workflows/test-snap-can-build.yml

```diff
@@ -11,16 +11,44 @@ permissions:
 jobs:
   build:
     runs-on: ubuntu-latest
-
     steps:
       - uses: actions/checkout@v3
 
-      - uses: snapcore/action-build@v1
+      - name: Setup snapcraft
+        env:
+          LP_BUILD_SECRET: ${{ secrets.LP_BUILD_SECRET }}
+          SNAPCRAFT_STORE_CREDENTIALS: ${{ secrets.SNAP_STORE_CANDIDATE }}
```

Comment:
We definitely don't want to expose this secret to pull requests, but don't worry, that secret is part of an environment that only triggers on builds from the `candidate` branch as an extra precaution. So even if this was using the `pull_request_target`, it still wouldn't work.